### PR TITLE
build game before dotnet project

### PR DESCRIPTION
**Pull request by bruno-garcia**
Build the Unity sample (installs packages required to build .NET) before building the .NET project

Prob not the final format since we dont' want this to run each time. 
Perhaps best to run after a Clean, or if we detect the unity packages haven't been restored yet.

Also useful for CI (won't need the manual step

- Somehow fails the first build. Unity builds seems to return before the test DLLs are available. The .NET build that kicks off right after the Unity's fail. But If I trigger it again right after, it works.
Seems like it worked on Windows in CI, but that's what I noticed on a mac (CI cache maybe?)

#skip-changelog

### CODE REVIEW

File: package-dev/Tests/Runtime/TestSentryOptions.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"enabled":true,"captureInEditor":true,"dsn":"http://test.com","debug":true,"debugOnlyInEditor":true,"diagnosticsLevel":3,"requestBodyCompressionLevel":1,"attachStacktrace":true,"sampleRate":1,"release":"release","environment":"test"}
```
Comment (bruno-garcia) [inline package-dev/Tests/Runtime/TestSentryOptions.json:1]: should be this be in gitignore?

**Comment by bruno-garcia** [comment]
@semuserable @bitsandfoxes thoughts?

File: package-dev/Tests/Runtime/TestSentryOptions.json (hunk 1 above)
Comment (bitsandfoxes) [reply package-dev/Tests/Runtime/TestSentryOptions.json:1]: If we ship the package pre-configured wouldn't that be a good thing?
Install Package -> Hit Play -> Events get already logged on our Unity-Project?

Comment (bruno-garcia) [reply package-dev/Tests/Runtime/TestSentryOptions.json:1]: This one seems to be only used on tests though (based on the name). I believe it's created during Play tests.

Comment (semuserable) [reply package-dev/Tests/Runtime/TestSentryOptions.json:1]: This is used in tests only, yes. Seems ok to be `.gitignore`d.

**Review by semuserable — APPROVED** [review]

**Pull request merged**